### DISCUSSION TITLE
hello i opend a issue a long time ago (about a year) how to use XPT2046 touch on a paralell lcd but still have some questions about that

### DISCUSSION THREAD

**Discussion by hannescam** [Bodmer/TFT_eSPI#2097] [Q&A - Touch] [answered]
hello i wanted to ask how to integrate a external touch library to tft_espi

**Comment by Bodmer** [comment, marked as answer]
The library does not support the use of a SPI touch controller when a parallel interface is used. You will have to find a suitable touch library and consult that libraries documentation. TFT_eTouch should work.

**Reply by hannescam** [reply]
could you also provide an example code for tft_etouch and a link where i can get it

**Comment by hannescam** [comment]
thankyou i am working on an minesweeper for esp32 for ili9341 or ili9348
it will be verry helpfull
